dataprofiler/tests/plugins/test_plugins.py

```diff
@@ -2,22 +2,44 @@
 from collections import defaultdict
 from unittest import mock
 
-from dataprofiler.plugins.__init__ import get_plugins
+from dataprofiler.plugins.__init__ import get_plugins, load_plugins
 from dataprofiler.plugins.decorators import plugin_decorator, plugins_dict
 
 
 class TestPlugins(unittest.TestCase):
-    def test_plugin_presets(self, *mocks):
+    def test_decorator_get_plugin(self, *mocks):
         mock_plugin_execution = mock.Mock()
         with mock.patch.dict(plugins_dict, defaultdict(dict)) as mock_plugin_dict:
 
-            @plugin_decorator(typ="test_preset", name="mock_test")
+            @plugin_decorator(typ="test", name="mock_test")
             def test_plugin():
                 mock_plugin_execution()
 
             expected_default_dict = defaultdict(dict)
-            expected_default_dict["test_preset"]["mock_test"] = test_plugin
+            expected_default_dict["test"]["mock_test"] = test_plugin
             self.assertDictEqual(expected_default_dict, mock_plugin_dict)
 
-            test_get_dict = get_plugins("test_preset")
+            test_get_dict = get_plugins("test")
             self.assertDictEqual({"mock_test": test_plugin}, test_get_dict)
+
+    @mock.patch("dataprofiler.plugins.__init__.importlib.util")
+    @mock.patch("dataprofiler.plugins.__init__.os.path.isdir")
+    @mock.patch("dataprofiler.plugins.__init__.os.listdir")
+    def test_load_plugin(self, mock_listdir, mock_isdir, mock_importlib_util):
+        mock_listdir.side_effect = (
+            lambda dir: ["__pycache__", "py"]
+            if dir.endswith("plugins")
```

Comment:
change `dir` var name to something 
- more descriptive 
- not a python specific / built-in variable name